plugins/plugin-optimize/README.md

```diff
@@ -38,5 +39,6 @@ Then add this plugin to your Snowpack config:
 | :--------------- | :-------: | :-------------------------------------------------------------- |
 | `minifyJS`       | `boolean` | Should JS be minified? (default: `true`)                        |
 | `preloadModules` | `boolean` | Should static `import`s be preloaded in HTML? (default: `true`) |
+| `target` | `string | string[]` | The language target(s) to transpile to. This can be a single string (ex: "es2018") or an array of strings (ex: ["chrome58","firefox57"]). If undefined, no transpilation will be done. See [esbuild documentation](https://github.com/evanw/esbuild) for more. |
```

Comment:
Not a request for change, but if you have `editor.formatOnSave: true` in VS Code settings, it’ll auto-align all the table columns (or you could run the “Format Document” command manually too). Doesn’t matter one bit here, but when there are a bunch of rows it can clean up mess.